Arelle/Arelle
# Fix deprecation warnings by making some strings with backslashes raw.

#### Steps to Test
Run `tests/integration_tests/validation/ESEF/test_esef_xhtml_conformance_suite.py`.  Make sure it passes and that the only deprecation warnings are about the `imp` module.

**review**:
@Arelle/arelle
